templates/_schemas/definitions/thing.twig

```diff
@@ -3,10 +3,15 @@
   
   {% set schema = schema ?? craft.schema.Thing %}
   {% set schema = schema
-    .name(props.title)
+    .name(props.name ?? props.title)
     .url(props.url|default(null))
+    .sameAs(props.website|default(null))
     .image(props.images|default([])|map(img => imageObject(img))|default(null))
-    .description(props.description|default('')|striptags|default(null))
+    .description(props.description|default('')|striptags|replace('&nbsp;', ' ')|default(null))
```

Comment:
We should double check if the `description` property is allowed to have HTML or not. In case it should have HTML (check Google’s recommendation) then we should not strip tags etc.

Reply:
@rungta the allowed property type for this one is _Text_.

<img width="949" alt="Screenshot 2023-01-11 at 5 43 32 PM" src="https://user-images.githubusercontent.com/30666052/211803577-08e1a6a1-062d-442a-926e-da62f43619bc.png">

Reply:
That is Schema.org’s spec. What about Google’s recommendation (for Articles)?

Reply:
@rungta I found the format as `HTML` only for _Job Posting_ description for all others it was plain text.